github-actions/DotNet.GitHubAction/DotNet.GitHubAction/Program.cs

```diff
@@ -79,7 +79,10 @@ static async Task StartAnalysisAsync(ActionInputs inputs, IHost host)
         {
             textWriter.WriteLine($"updated-metrics={updatedMetrics}");
             textWriter.WriteLine($"summary-title={title}");
-            textWriter.WriteLine($"summary-details={summary}");
+            textWriter.WriteLine(
+                "summary-details<<EOF\n" +
+                summary +
+                "\nEOF\n" );
```

Comment:
Just nit picking on style 😁 

I would think it would be better to do multiple write lines instead of a single with multiple lines, or use the interpolated string. This one is very clear as to the intent of the output:
```csharp
textWriter.WriteLine("summary-details<<EOF");
textWriter.WriteLine(summary);
textWriter.WriteLine("EOF");
```
or this, which is less clear but a one-liner:
```csharp
textWriter.WriteLine($"summary-details<<EOF\n{summary}\nEOF\n");
```
@IEvangelist ping

Reply:
Sure thing. Updated to use multiple `testWriter.WriteLine`. Let me know if there is anything else.